app/views/student_quizzes/take_quiz.html.erb

```diff
@@ -14,6 +14,11 @@
 
   <%= hidden_field_tag :questionnaire_id, "#{@questionnaire.id}" %>
   <%= hidden_field_tag :assignment_id, "#{params[:assignment_id]}" %>
-  <%= render :partial => 'quiz_form', :locals => { :questionnaire_id => @questionnaire.id,:display=>"take" } %>
```

Comment:
I didn't see you commit the changes on quiz_form partial. Is this partial still used? If not, remove it.